interfaces/IPriceOracle.json

```diff
@@ -3,42 +3,242 @@
   "contractName": "IPriceOracle",
   "sourceName": "contracts/interfaces/IPriceOracle.sol",
   "abi": [
-    {
-      "inputs": [
-        {
-          "internalType": "address",
-          "name": "asset",
-          "type": "address"
-        }
-      ],
-      "name": "getAssetPrice",
-      "outputs": [
-        {
-          "internalType": "uint256",
-          "name": "",
-          "type": "uint256"
-        }
-      ],
-      "stateMutability": "view",
-      "type": "function"
-    },
-    {
-      "inputs": [
-        {
-          "internalType": "address",
-          "name": "asset",
-          "type": "address"
-        },
-        {
-          "internalType": "uint256",
-          "name": "price",
-          "type": "uint256"
-        }
-      ],
-      "name": "setAssetPrice",
-      "outputs": [],
-      "stateMutability": "nonpayable",
-      "type": "function"
+		{
```

Comment:
What's going on with the whitespace in this file?

Reply:
Removed changes to `IPriceOracle.json` that belong in `AaveOracle.json`.